resources/arduino_files/telemetry_board/telemetry_board.ino

```diff
@@ -107,6 +112,22 @@ class BaseNameHandler {
       Serial.print("\":");
     }
 
+    bool NameEquals(const char* s, uint8_t len) {
```

Comment:
It's not all intuitive to me what is going on here and since this is in an actual file people might look at (as opposed to the shared files) maybe some comments or description of what is happening.

Reply:
Oh, good idea.